ddpui/api/dbt_api.py

```diff
@@ -70,6 +70,19 @@ def put_dbt_github(request, payload: OrgDbtGitHub):
     org.dbt.gitrepo_access_token_secret = payload.gitrepoAccessToken
     org.dbt.save()
 
+    if payload.gitrepoAccessToken is not None and payload.gitrepoAccessToken != "":
+        gitrepo_url = payload.gitrepoUrl.replace(
```

Comment:
we need to create the secret block if the token is passed and the block does not exist

Reply:
good catch, i have changed the edit into an upsert please see

Reply:
we also create the secret block in our db in `OrgPrefectBlockv1`. We should do a upsert in that django table also

Reply:
ok done